### DISCUSSION TITLE
Renaming diabetes mellitus templates

### DISCUSSION THREAD

**Discussion by lamurian** [bgsi-registry/OpenEHR-registry-template#28] [Ideas]
# Background

As per #17, I'm proposing renaming the DM templates to make them more concise. As I evaluated the template, I saw that there's several inconsistencies between template name, `templateId`, and the semver. Thus, I request renaming these templates to prevent issues with the backend. The file names I'm proposing are as follow:

# DM Dewasa:

- `diabetesMellitus-vascularSurgery-v0`
- `diabetesMellitus-endocrinology-v0`
- `diabetesMellitus-gastroEnteroHepatology-v0`
- `diabetesMellitus-neurologi-v0`
- `diabetesMellitus-physExam-v.0`
- `diabetesMellitus-nephrologyHypertension-v0`
- `diabetesMellitus-cardiology-v0`
- `diabetesMellitus-ophthalmology-v0`
- `diabetesMellitus-psychiatry-v0`
- `diabetesMellitus-otolaringology-v0`
- `diabetesMellitus-urologyMale-v0`
- `diabetesMellitus-urologyFemale-v0`

# DM Anak

- `diabetesMellitusAdolescent-vascularSurgery-v0`
- `diabetesMellitusAdolescent-endocrinology-v0`
- `diabetesMellitusAdolescent-gastroEnteroHepatology-v0`
- `diabetesMellitusAdolescent-neurologi-v0`
- `diabetesMellitusAdolescent-physExam-v.0`
- `diabetesMellitusAdolescent-nephrologyHypertension-v0`
- `diabetesMellitusAdolescent-cardiology-v0`
- `diabetesMellitusAdolescent-ophthalmology-v0`
- `diabetesMellitusAdolescent-psychiatry-v0`
- `diabetesMellitusAdolescent-otolaringology-v0`
- `diabetesMellitusAdolescent-urologyMale-v0`
- `diabetesMellitusAdolescent-urologyFemale-v0`
- `diabetesMellitusAdolescent-social-v0`

CC: dok @mnardisasmita

**Comment by mnardisasmita** [comment]
ok dok @lamurian , I will make a new version with new name convention soon

**Reply by lamurian** [reply]
Thank you dok. This is our least priority atm, so I'll just leave it here. After we finished all dev iterations, I'll issue a ticket and assign it to you dok. For the moment, you should focus on drafting the DMD template (just make sure to keep the name, `templateId`, and semver consistent).
